lib/cron.dart

```diff
@@ -130,6 +130,25 @@ class Schedule {
       seconds != null &&
       seconds!.isNotEmpty &&
       (seconds!.length != 1 || !seconds!.contains(0));
+
+  /// Converts the schedule into a cron-formatted string.
+  String toCronString() {
+    final minutesStr = _convertToCronString(minutes);
+    final hoursStr = _convertToCronString(hours);
+    final daysStr = _convertToCronString(days);
+    final monthsStr = _convertToCronString(months);
+    final weekdaysStr = _convertToCronString(weekdays);
+
+    return '$minutesStr $hoursStr $daysStr $monthsStr $weekdaysStr';
```

Comment:
Could you please also handle the seconds here too (if they are not null)?

Reply:
At the beginning, I had put seconds, but sites like [crontab](https://crontab.guru/) had problems with the created format. After researching, I found[[1](https://en.wikipedia.org/wiki/Cron),[2](https://www.ibm.com/docs/en/db2oc?topic=task-unix-cron-format)] out that the standard cron is five parts. That's why I deleted the second. Are you sure to add seconds?

Reply:
It is non-standard, but the internals and the parsing supports it anyway, so I think `toCronString` should too, otherwise it would be hard to do round-trip tests. (object -> string -> object roundtrips anyway, as string -> object -> string may not be feasible with the lack of canonical representation).

Reply:
What do you think if this topic is optional?

Reply:
I think it should be optional indeed based on `_hasSeconds`.